api/v1beta1/events.go

```diff
@@ -73,95 +74,95 @@ var Events = map[string]DisruptionEvent{
 		OnTargetTemplateMessage:        "pod seems to have recovered from the disruption %s failure",
 		Category:                       TargetEvent,
 	},
 	EventNodeMemPressureState: {
 		Type:                           corev1.EventTypeWarning,
 		Reason:                         EventNodeMemPressureState,
-		OnDisruptionTemplateMessage:    "targeted node %s is under memory pressure",
-		OnDisruptionTemplateAggMessage: "targeted node(s) are under memory pressure",
+		OnDisruptionTemplateMessage:    "Targeted node %s is under memory pressure",
+		OnDisruptionTemplateAggMessage: "Targeted node(s) are under memory pressure",
 		OnTargetTemplateMessage:        EventOnTargetTemplate + "node is under memory pressure",
 		Category:                       TargetEvent,
 	},
 	EventNodeDiskPressureState: {
 		Type:                           corev1.EventTypeWarning,
 		Reason:                         EventNodeDiskPressureState,
-		OnDisruptionTemplateMessage:    "targeted node %s is under disk pressure",
-		OnDisruptionTemplateAggMessage: "targeted node(s) are under disk pressure",
+		OnDisruptionTemplateMessage:    "Targeted node %s is under disk pressure",
+		OnDisruptionTemplateAggMessage: "Targeted node(s) are under disk pressure",
 		OnTargetTemplateMessage:        EventOnTargetTemplate + "node is under disk pressure",
 		Category:                       TargetEvent,
 	},
 	EventNodeUnavailableNetworkState: {
 		Type:                           corev1.EventTypeWarning,
 		Reason:                         EventNodeUnavailableNetworkState,
-		OnDisruptionTemplateMessage:    "targeted node %s network is unavailable",
-		OnDisruptionTemplateAggMessage: "targeted node(s) network are unavailable",
+		OnDisruptionTemplateMessage:    "Targeted node %s network is unavailable",
+		OnDisruptionTemplateAggMessage: "Targeted node(s) network are unavailable",
 		OnTargetTemplateMessage:        EventOnTargetTemplate + "node network is unavaialble",
 		Category:                       TargetEvent,
 	},
 	EventNodeWarningState: {
 		Type:                           corev1.EventTypeWarning,
 		Reason:                         EventNodeWarningState,
-		OnDisruptionTemplateMessage:    "targeted node %s is not ready",
-		OnDisruptionTemplateAggMessage: "targeted node(s) are not ready",
+		OnDisruptionTemplateMessage:    "Targeted node %s is not ready",
+		OnDisruptionTemplateAggMessage: "Targeted node(s) are not ready",
 		OnTargetTemplateMessage:        EventOnTargetTemplate + "node is not ready",
 		Category:                       TargetEvent,
 	},
 	EventNodeRecoveredState: {
 		Type:                           corev1.EventTypeNormal,
 		Reason:                         EventNodeRecoveredState,
-		OnDisruptionTemplateMessage:    "targeted node %s seems to have recovered",
-		OnDisruptionTemplateAggMessage: "targeted node(s) seem to have recovered",
-		OnTargetTemplateMessage:        "node seems to have recovered from the disruption %s failure",
```

Comment:
Unifying the messages for all event to have an upper case at the beginning. This is the format of Kubernetes